src/languages/cs.js

```diff
@@ -134,15 +137,23 @@ function(hljs) {
         ]
       },
       {
+        // [Attributes("")]
+        className: 'title',
+        begin: '^\\s*\\[', excludeBegin: true, end: '[\\(\\]]', excludeEnd: true,
+        contains: [
+          STRING
+        ]
+      },
+      {
         // Expression keywords prevent 'keyword Name(...)' from being
         // recognized as a function definition
         beginKeywords: 'new return throw await',
         relevance: 0
       },
       {
         className: 'function',
-        begin: '(' + TYPE_IDENT_RE + '\\s+)+' + hljs.IDENT_RE + '\\s*\\(', returnBegin: true, end: /[{;=]/,
-        excludeEnd: true,
+        begin: '(' + TYPE_IDENT_RE + '\\s+)+[A-Z]\\w*\\s*\\(', returnBegin: true,
+        end: /[{;=]/, excludeEnd: true,
```

Comment:
Looks like this change broke this use case:

    int[] f(int val = 0) { .. }

It's not recognized as a function declaration anymore, as it does in master.